.release/linux/package/usr/lib/systemd/system/openbao.service

```diff
@@ -28,7 +27,7 @@ Restart=on-failure
 RestartSec=5
 TimeoutStopSec=30
 LimitNOFILE=65536
-LimitMEMLOCK=infinity
+MemorySwapMax=0
```

Comment:
So this would only work for native (non-containerized) installation... For the latter, would we recommend running with `--memory-swappiness=0` on the container execution CLI? [IIUC](https://docs.docker.com/config/containers/resource_constraints/#--memory-swappiness-details) that seems to do what we want? 

If so, we'd probably want to mention it in the various places where `mlock` was previously mentioned.

Reply:
Oh yeah, I didn't think about the Docker case... That flag looks right. I'll add it to the "Post-installation hardening" section. If you're talking about the other places in the docs where mlock was mentioned, I don't want to replace them in this commit, because frankly I thought they were a bit random. I will work on a bigger docs reorg, and maybe add more warnings about swap then.

Reply:
Ah, I was thinking about perhaps in the container entrypoint scripts/Dockerfiles?

Reply:
I don't think you can specify this on the Dockerfile, nor in the entrypoint script _within the container_. Do we provide a docker-compose template? If we did we could do it there...

Reply:
Oh, sorry, no, but we had commentary e.g. here:

https://github.com/openbao/openbao/blob/ee45b4d893832f0184c669f65ae8eed125548f4f/Dockerfile#L63-L64

that mentions mlock, so I was thinking perhaps suggesting that the caller use these options outside of the containerfile?